nrich-registry/README.md

```diff
@@ -4,19 +4,22 @@
 
 ## Overview
 
-nrich-registry is a library whose purpose is to make editing of registry entities from client side easier. It transforms JPA entities in a format that client can interpret to create dynamic forms and
-grids for editing of entities without additional implementation on server side. Library provides REST API for searching, creating, updating and deleting entities. Users are only required to provide
+`nrich-registry` is a module whose purpose is to make editing of registry entities from client side easier. It transforms JPA entities in a format that client can interpret to create dynamic forms
```

Comment:
Maybe change first sentence into something like `Main purpose of nrich-registry module is to make administration of registry entities on client side easier.`

Reply:
I would keep the module name first since that seems to be the same as in other `core` modules but will change the rest.